packages/docs/docs/guide/getting-started.md

```diff
@@ -27,43 +30,49 @@ You will then have the opportunity to configure your VuePress site’s metadata
 
 Once it is complete, you should see your new VuePress site scaffolded in your directory! :tada:
 
-## Manual Setup
+At this time, you'll need to take one more step and update the VuePress version inside the generated project.
+
+```bash
+# Navigate into your newly created project
+cd [directoryName]
+
+# Update VuePress dependency
+yarn add -D vuepress # npm install --save-dev vuepress
```

Comment:
Is this necessary? I think it's already included in the template, and the newest version will be installed.

Reply:
@meteorlxy I'd prefer not to include this to be honest. 

It currently only installs 1.3.2. I don't have npm permissions to publish the new update. https://github.com/vuepressjs/create-vuepress/pull/19

Do you have access to fix this?